docs/source/conf.py

```diff
@@ -485,26 +485,26 @@ def autodoc_process_docstring(app, what, name, obj, options, lines):  # noqa: PL
     remove_arrows_in_examples(lines)
 
 
-def env_override(default_appid):
-    build_version = os.getenv("READTHEDOCS_VERSION")
+# def env_override(default_appid):
```

Comment:
Is there a reason for this chunk of code to be commented out rather than removed?

Reply:
i think we need to uncomment it before we merge. This was only to get the forked version running.